gems/smithy-client/lib/smithy-client/stubbing/data_applicator.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Client
+    module Stubbing
+      # @api private
+      class DataApplicator
+        include Smithy::Schema::Shapes
+
+        def initialize(rules)
+          @rules = rules
+        end
```

Comment:
`@rules` are technically shapes now?

Reply:
Yeah, well this is technically service shape or schema.. we can consider changing all of the rules lingo to schema or use rules.